### ISSUE TITLE
Fatal Exception: java.lang.IllegalStateException Fragment already added: (tag=StorylyDialogFragment)

### ISSUE THREAD

**Issue by yasincidem** [Netvent/storyly-mobile#134]
## Environment Details

**Storyly SDK Version:**   1.15.1
**Android SDK Version:**     Android SDK Independent
**Device Model:**  Device Model Independent
**Device OS Version:**   OS version independent

App is in foreground when this crash occurs.

```
Fatal Exception: java.lang.IllegalStateException: Fragment already added: q{85a25e4} (e25964d6-8e83-4a0b-9c29-1394449c47e9 tag=StorylyDialogFragment)
       at androidx.fragment.app.FragmentStore.addFragment(FragmentStore.java:92)
       at androidx.fragment.app.FragmentManager.addFragment(FragmentManager.java:1367)
       at androidx.fragment.app.BackStackRecord.executeOps(BackStackRecord.java:387)
       at androidx.fragment.app.FragmentManager.executeOps(FragmentManager.java:1851)
       at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:1759)
       at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:1703)
       at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1646)
       at androidx.fragment.app.FragmentManager$4.run(FragmentManager.java:480)
       at android.os.Handler.handleCallback(Handler.java:790)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:164)
       at android.app.ActivityThread.main(ActivityThread.java:7000)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:441)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1408)
```

Is this a know issue or a new now? What could be the fix to this crash?

**Comment by levent-oral** [comment]
Duplicate of [#132](https://github.com/Netvent/storyly-mobile/issues/132)

**Comment by levent-oral** [comment]
Hi @yasincidem ,

We've reproduced and fixed this issue in [Android SDK 1.17.0 version](https://github.com/Netvent/storyly-mobile/blob/master/android/RELEASENOTES.md#1170-21102021). Please check with latest version. 

Thanks,

**Comment by yasincidem** [comment]
Upgrading to 1.17.0 didn't actually fix the crash but adding android:splitMotionEvents="false" to StorylyView in XML fixed it.
